Vagrantfile

```diff
@@ -11,7 +11,7 @@ CONFIG = File.join(File.dirname(__FILE__), "config.rb")
 # Defaults for config options defined in CONFIG
 $num_instances = 1
 $instance_name_prefix = "core"
-$update_channel = "alpha"
```

Comment:
I'll admit I feel a little weird about this. Why is $update_channel the one special thing we allow to be set using an env var?